indexing-service/src/test/java/org/apache/druid/indexing/common/task/RangePartitionCachingLocalSegmentAllocatorTest.java

```diff
@@ -141,6 +142,16 @@ public void allocatesCorrectShardSpecsForLastPartition()
     testAllocate(row, interval, partitionNum, null);
   }
 
+  @Test
+  public void test_getSequenceName_forIntervalAndRow_shouldUseISOFormatAndPartitionNumForRow()
```

Comment:
Thanks for adding tests. I think we prefer to use a short but intuitive name for tests and add comments if necessary.

Reply:
can you recommend a name? I like this format because it's explicit about what the test is doing

test_<functionUnderTest>_<condition>_<expectedResult>

Reply:
Hmm, I would just use `getSequenceName()` for these tests with a comment for the expected value because they look simple enough to understand.

> can you recommend a name? I like this format because it's explicit about what the test is doing
> 
> `test_<functionUnderTest>_<condition>_<expectedResult>`

I agree on that we could have a better format or a better practice for test names (we've been talking about a better practice for testing [here](https://www.mail-archive.com/dev@druid.apache.org/msg02426.html)). Unfortunately, we don't have such a format or practice and I think it could be better to make them consistent for now because we don't know what kind of policy we would adopt yet. I think it would be worth to start a discussion on it.

Reply:
Ok I've renamed the tests to the less explicit format and left a comment for how I'd rename them in the future. When the tests pass, can you merge?

How do I start this discussion? The linked discussion is about a lot of different testing practices and I think driving consensus on such a large topic will take a long time. How do I start a discussion thread about naming practices. Who needs to approve the change to the naming format? And who has veto rights?

Reply:
Sorry for late reply. You can start a discussion by starting a new thread on dev which is you already did (thanks!). There's no approval or veto in this kind of discussion. Anyone can participate in the discussion and express their opinion. The discussion could be concluded when the community accepts it.